File: backend/app/src/main/java/com/onair/hearit/keyword/dto/KeywordResponse.java
package com.onair.hearit.keyword.dto;

import com.onair.hearit.domain.Keyword;

public record KeywordResponse(
        Long id,
        String name
) {
    public static KeywordResponse from(Keyword keyword) {
        return new KeywordResponse(
                keyword.getId(),
                keyword.getName()
        );
    }
}

File: android/app/src/main/java/com/onair/hearit/data/dto/KeywordResponse.kt
package com.onair.hearit.data.dto

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class KeywordResponse(
    @SerialName("id")
    val id: Long,
    @SerialName("name")
    val name: String,
)
